packages/web-react/src/components/Accordion/README.md

```diff
@@ -198,6 +198,12 @@ On top of the API options, the components accept [additional attributes][readme-
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
+## Dependencies
```

Comment:
**suggestion (if-minor):** I would suggest using simply the title `Icons`. And reword is as suggested by @crishpeen. The user/developer does not care what is used internally. The main information here is that the component is using icons and we want to refer him/her to the right place. :-)

Reply:
I second that.

Reply:
edited